src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/CompilerServices/OptionTextAttribute.vb

```diff
@@ -14,7 +14,7 @@ Namespace Global.Microsoft.VisualBasic.CompilerServices
     ''' WARNING: Do not rename this attribute or move it out of this 
     ''' module.  Otherwise there are compiler changes that will
     ''' need to be made!
-    ''' </remarsk>
+    ''' </remarks>>
```

Comment:
>>> [](start = 17, length = 2)

Double `>`.

Reply:
Someone else's code, I removed > that code added when I corrected the spelling.